src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -107,6 +164,34 @@ public function action_handle_verify_webhooks() : void {
 		$this->container->make( Webhooks::class )->handle_verification();
 	}
 
+	/**
+	 * Handles the setting up of the webhook on the settings page.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function action_handle_set_up_webhook(): void {
+		$nonce  = tribe_get_request_var( 'tc_nonce' );
+		$status = esc_html__( 'Something went wrong with your Webhook Creation. Please try again later.', 'event-tickets' );
+
+		$webhooks = $this->container->make( Webhooks::class );
+
+		if ( ! wp_verify_nonce( $nonce, $webhooks::NONCE_KEY_SETUP ) || ! current_user_can( \Tribe\Admin\Pages::get_capability() ) ) {
+			wp_send_json_error( [ 'status' => $status ] );
```

Comment:
It's either a nonce failure or a permission failure. I didn't want to become more specific since to land there the most likely reason is system abuse.

I have added `Please reload the page and try again later.` in order to help the users that have actually left the page open so long that the nonce expired.